_includes/scenarios/integrate/share-own-attribute-to-peer.md

```diff
@@ -1,11 +1,285 @@
+
+| Attribute Type | Attribute Owner | Possible? | Automation                                                     | Examples/Reason                                                                                                                                                                                                                                                                                        |
+| -------------- | --------------- | :-------: | -------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Identity       | Sender          |     ✓     | `AUTO ACCEPT`                                                  | Company sends new customer the address of the company.                                                                                                                                                                                                                                                 |
+| Identity       | Recipient       |     ✗     | `N/A`                                                          | It makes no sense to share the Attribute with the Recipient, because he already owns it.                                                                                                                                                                                                               |
+| Identity       | Third Party     |     ✗     | `N/A`                                                          | You cannot share an Attribute of which you are not the owner.                                                                                                                                                                                                                                          |
+| Identity       | `<empty>`       |     ✓     | `AUTO ACCEPT`                                                  | An empty owner defaults to an Attribute with `owner=<Sender>`.                                                                                                                                                                                                                                         |
+| Relationship   | Sender          |     ✓     | `USER DECISION` / `NOT ALLOWED` (depending on confidentiality) | A user can share own RelationshipAttributes of any Relationship with any other Relationship (if the confidentiality of the RelationshipAttribute is protected or public).<br>Example: Share customer ID from company A with company B (User is owner of RelationshipAttribute).                        |
```

Comment:
I suppose we should make a distinction here, too. It seems to me that the description here applies to the case, that the Relationship owned by the Sender exists in the context of a Relationship with a ThirdParty, not with the Recipient. Otherwise, the Recipient would already have the RelationshipAttribute and re-sharing it would make no sense. (We should also check if the implementation resembles this case.)

Reply:
It is currently possible to re-share the RelationshipAttribute, which exists between the Sender and Recipient and is owned by the Sender. This probably doesn't make sense, but I think it's better to keep the documentation vague for the time being and correct the implementation first and then adapt the documentation accordingly. If we now make the documentation more precise without already having the implementation, the documentation and implementation would no longer match at all